configs/local/data/license/LiferayActivationKey_DXPDevelopment_developer.li

```diff
@@ -1 +1 @@
-rO0ABXc0AAAABAAWTGlmZXJheSBBY3RpdmF0aW9uIEtleQAWTGlmZXJheSBBY3RpdmF0aW9uIEtleXNyAA5qYXZhLnV0aWwuRGF0ZWhqgQFLWXQZAwAAeHB3CAAAAZQclk4AeHVyABNbTGphdmEubGFuZy5TdHJpbmc7rdJW5+kde0cCAAB4cAAAAAB1cQB+AAIAAAAAegAAAS8BCDZjNTI0ZGU5ZWZjYjU0YWYwMjlkODVjMGQwYzQzOWM1NDk4NGYzYzI0NzY1Njk3ZWMwZDMxODQ1ZjVlY2EwYWQzMzA0ZjBjZGE1ZWZmYTNmMzYwYThlNDcwMDNkNmVmOGU2NGNkNDYxZDdhMzFhOTNkNjAzYjNmNTAyZDk2MmVmNmM4MDkxZDhiNjYxNDc0MjY1OTllZDk1ODdmZTE4ZDlkZGZlN2JiZjM3NmY3NjBlOTgzOWMyN2I1NDIzMjlkMWU2ZDk0NjE5MGM5MWJiYjViM2IyZmYyZGU3YWZhMTYwYzIwYThlZmJlYmM1OTAxNGVjN2ExNzA2MmI0NDRlYzY4NjEzYjVmNAAAAZCiM26dAA1EWFAgRGV2ZWxvcGVyAAlkZXZlbG9wZXIAATV1cQB+AAIAAAAAd1AAAAAKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWTGlmZXJheSBBY3RpdmF0aW9uIEtleQAPRFhQIERldmVsb3BtZW50AAZQb3J0YWwAAzcuNHVxAH4AAgAAAABzcQB+AAB3CAAAAYzE5SIAeA==
\ No newline at end of file
+rO0ABXc0AAAABAAWTGlmZXJheSBBY3RpdmF0aW9uIEtleQAWTGlmZXJheSBBY3RpdmF0aW9uIEtleXNyAA5qYXZhLnV0aWwuRGF0ZWhqgQFLWXQZAwAAeHB3CAAAAZQclk4AeHVyABNbTGphdmEubGFuZy5TdHJpbmc7rdJW5+kde0cCAAB4cAAAAAB1cQB+AAIAAAAAegAAAS8BCDZjNTI0ZGU5ZWZjYjU0YWYwMjlkODVjMGQwYzQzOWM1NDk4NGYzYzI0NzY1Njk3ZWMwZDMxODQ1ZjVlY2EwYWQzMzA0ZjBjZGE1ZWZmYTNmMzYwYThlNDcwMDNkNmVmOGU2NGNkNDYxZDdhMzFhOTNkNjAzYjNmNTAyZDk2MmVmNmM4MDkxZDhiNjYxNDc0MjY1OTllZDk1ODdmZTE4ZDlkZGZlN2JiZjM3NmY3NjBlOTgzOWMyN2I1NDIzMjlkMWU2ZDk0NjE5MGM5MWJiYjViM2IyZmYyZGU3YWZhMTYwYzIwYThlZmJlYmM1OTAxNGVjN2ExNzA2MmI0NDRlYzY4NjEzYjVmNAAAAZFRwEvLAA1EWFAgRGV2ZWxvcGVyAAlkZXZlbG9wZXIAATV1cQB+AAIAAAAAd1AAAAAKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWTGlmZXJheSBBY3RpdmF0aW9uIEtleQAPRFhQIERldmVsb3BtZW50AAZQb3J0YWwAAzcuNHVxAH4AAgAAAABzcQB+AAB3CAAAAYzE5SIAeA==
```

Comment:
Is this change required?

Reply:
Or the serverId one?

Reply:
Unsolicited. I reverted to the previous version.